src/Pages/OtherProjects/OtherProjects.css

```diff
@@ -35,6 +35,16 @@
     height: 300px;
 }
 
+.TheLastOfUsEpContainer button.carouselButton:nth-child(3) + div {
```

Comment:
i'm curious if you could target both carousels with just `button.carouselButton:nth-child(3) + div` ??

Reply:
looks like it needs the ep container as well  to work

Reply:
I thought it might. Wasn't sure though. Sounds good